File: jsprobelms/day10/count-dup.js
// count the number of duplicates present  in an array
let arr=[1,1,2,5,5,2,5,2,4,5,2,1,5,8,7,22,26,8,7]
res=[]
dup=[]
for(i of arr){
    if(!res.includes(i)){
        res.push(i)
    }
    else{
        dup.push(i)
    }
}
console.log(res,dup);
res.sort((a,b)=>a-b)
for(j of dup){
    count=0
    for(k of arr){
        if(j==k){
            count++
        }
    }
    console.log(j,"is present",count,"times");
}

File: pythonproblems/day10/count-dup.py
# count duplicates in array
arr=[1,2,4,53,2,1,4,5,3,2,1,5,3]
res=[]
dup=[]
for i in arr:
   
    if i not in res:
        res.append(i)
    else:
        dup.append(i)
for j in dup:
    count=0
    for k in arr:
        if j==k:
            count+=1
        
    print(j,"is present",count,"times")